posts/unified-memory-oversubscription/README.md

```diff
@@ -0,0 +1,31 @@
+# Unified Memory Oversubscription
+
+Benchmark for UVM oversubscription tests
+
+Build command: `nvcc uvm_oversubs.cu -gencode arch=compute_70,code=sm_70 -o uvm_oversubs`
+## Command line options
+
+```
+-m - UVM mode
+page_fault/zero_copy/prefetch_once_and_hints/stripe_gpu_cpu
+Default: page_fault
+
+-a - Access Pattern
+streaming/block_streaming/random_warp
+Default: streaming
+
+-p - Oversubscription factor
+Float value.
+Eg: 1.1 - 110% GPU allocation
+Default: 1.0
+
+-s - Page Size
```

Comment:
Hmm, what does this control? Maybe clarify it's not changing HW page size, but how the data is striped into pages in SW.

Reply:
Page size of UVM striping, this is not hardware page size. Will clarify in documents.